app/models/user.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const user = sequelize.define(
+    'user',
+    {
+      name: DataTypes.STRING,
```

Comment:
You should define in the model the properties of each field if it is unique, if it is optional, if it is a primary key etc.